app/controllers/concerns/curation_concerns/single_use_links_viewer_controller_behavior.rb

```diff
@@ -0,0 +1,64 @@
+module CurationConcerns
+  module SingleUseLinksViewerControllerBehavior
+    extend ActiveSupport::Concern
+    include CurationConcerns::DownloadBehavior # was Sufia::Download...
+    included do
+      include ActionDispatch::Routing::PolymorphicRoutes
+
+      skip_before_action :load_file, except: :download
+      rescue_from CurationConcerns::SingleUseError, with: :render_single_use_error
+      rescue_from CanCan::AccessDenied, with: :render_single_use_error
+      rescue_from ActiveRecord::RecordNotFound, with: :render_single_use_error
+      class_attribute :presenter_class
+      self.presenter_class = CurationConcerns::GenericFilePresenter
```

Comment:
I think that the presenter-based stuff (L12, 13, 25, 34-36) can be removed since it's not needed for the the single-use show view.